skimage/feature/_hog.py

```diff
@@ -276,6 +282,12 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
 
     n_blocks_row = (n_cells_row - b_row) + 1
     n_blocks_col = (n_cells_col - b_col) + 1
+    if n_blocks_col <= 0 or n_blocks_row <= 0:
+        min_row = (-1 + b_row) * c_row + 1
+        min_col = (-1 + b_col) * c_col + 1
+        raise ValueError('The image given is not large enough for the cells_per_block '
+                         'and pixels_per_cell given, should be at least:'
+                         f' {min_row} rows x {min_col} cols')
```

Comment:
I think this can't be right. For 
```python
import numpy as np
import skimage
img = np.zeros((17, 17))
skimage.feature.hog(img)
```
it still get the error telling me `should be at least: 17 rows x 17 cols` which `img` satisfies...